spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,21 @@
+require 'beaker-rspec'
+require 'beaker-puppet'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+install_module
+install_module_dependencies
+
+RSpec.configure do |c|
+  # Configure all nodes in nodeset
+  c.before :suite do
+    hosts.each do |host|
+      case fact('os.family')
+      when 'Debian'
+        #docker image does not provide cron
+        on(host, 'apt install cron --yes', acceptable_exit_codes: [0]).stdout
```

Comment:
There's a shorthand for this: `host.install_package('cron')`

Reply:
nice.
updated.